file_paths.md

```diff
@@ -121,14 +126,22 @@ The files annotated by an '*' indicate that they are created by package installa
         ssl *
         VERSION                           # puppet-agent package version
 
+    /opt/puppetlabs/pxp-agent *
+        modules *                         # contains pxp-agent modules (ruby binaries)
+            pxp-module-puppet *           # executes Puppet and provides results of Puppet runs
```

Comment:
Same comment as below - is this installed by packaging or does it come down after the puppet-agent run?